qiling/debugger/qdb/context.py

```diff
@@ -68,7 +73,7 @@ def try_read_pointer(self, address: int) -> Optional[bytes]:
         try to read pointer size of data from ql.mem
         """
 
-        return self.try_read(address, self.ql.pointersize)
+        return self.try_read(address, self.pointersize)
```

Comment:
I think that the size of the read should be the longest possible instruction for that specific arch (i.e. 4 for ARM, 15 for Intel, etc.)